oneconfig/block-lines-antialiasing.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Fix for NVidia Graphics or antialiasing issues.
+---
+
+# Problematic Block Lines - NVidia Antialiasing Issue
+
+If the user sees lines on blocks, they most likely have an NVidia GPU or have antialiasing on.(which if so, just turn it off)
```

Comment:
spacing on `on.(which`

Reply:
Also this isn't strictly a OneConfig issue... this should be in a separate category alongside some of the other fixes here